dashboard/views.py

```diff
@@ -297,6 +268,20 @@ def assignment_view(request, course_id=0):
     df.drop(columns=['assignment_id', 'due_date'], inplace=True)
     df.drop_duplicates(keep='first', inplace=True)
 
+    df3 = df[df['towards_final_grade']>0.0]
+    df3[['score']] = df3[['score']].astype(float)
+    df3['percent_gotten']=df3.apply(user_percent,axis=1)
```

Comment:
```suggestion
    df3['percent_gotten'] = df3.apply(user_percent, axis = 1)
```

Reply:
I'm not going to reformat all of these, just a few here.

Reply:
Looks like equals are supposed to be spaced out between an assignment or binary operators but not  in arguments.

Reply:
Is there an auto-way to do the reformatting job from IDE?

Reply:
Yes, I fixed it with IDE help.